File: benchmark/Python 2.7/百万素数查找耗时.py
import time
import math

total = 0
def isprime(num):
	global total
	count = int(math.sqrt(num))+1
	for i in xrange(2, count):
		if (num % i == 0):
			return 0
	total+=1
	return 1

start = time.time()
	
for i in xrange(0, 1000000):
	isprime(i)
print(total)

print(time.time()-start)

File: benchmark/PHP 7.0/百万素数查找耗时.php
<?php

$total = 0;
function isprime(int $num): int {
	global $total;
	$count = (int) sqrt($num) + 1;
	for ($i = 2; $i < $count; ++$i) {
		if ($num%$i==0)
			return false;
	}
	$total++;
	return true;
}

$starttime = microtime(true);

for ($i = 0; $i < 1000000; $i++)
	isprime($i);
echo $total, "\n";

echo microtime(true)-$starttime."\n";